mountpoint-s3-fs/src/prefetch.rs

```diff
@@ -62,7 +63,10 @@ use task::RequestTask;
 #[derive(Debug, Error)]
 pub enum PrefetchReadError<E> {
     #[error("get object request failed")]
-    GetRequestFailed(#[source] ObjectClientError<GetObjectError, E>),
+    GetRequestFailed {
+        source: ObjectClientError<GetObjectError, E>,
```

Comment:
Is the `#[source]` attribute not required here? Will `thiserror` just rely on the name of the field?

Reply:
right, as mentioned in [docs](https://docs.rs/thiserror/latest/thiserror/#details):

> The Error trait’s source() method is implemented to return whichever field has a #[source] attribute or is named source, if any.